packages/entity-list/src/components/AdminSearchForm/AdminSearchForm.js

```diff
@@ -60,6 +60,11 @@ const AdminSearchForm = ({resetSearch, intl, searchFilters}) => {
         onClick={resetSearch}
         title={msg('client.entity-list.reset')}
       />
+      <BallMenu buttonProps={{icon: 'ellipsis-h'}}>
+        <MenuItem onClick={saveSearchFilter}>
```

Comment:
Bonus Quest: You could disable the button if the search form has no user made changes.

Bind dirty prop in container:

`import {isDirty} from 'redux-form'`
...
` searchFormDirty: isDirty('searchForm')(state)`

And then set disabled property: 

` <MenuItem disabled={!searchFormDirty} onClick={saveSearchFilter}>`